### ISSUE TITLE
No email returned

### ISSUE THREAD

**Issue by ndvbd** [facebookarchive/php-graph-sdk#1068]
When I do this:

https://graph.facebook.com/USER_ID/permissions?access_token=THE_TOKEN

I get:

```
{
   "data": [
      {
         "permission": "email",
         "status": "granted"
      },
      {
         "permission": "public_profile",
         "status": "granted"
      }
   ]
}
```

However when I do:

`https://graph.facebook.com/USER_ID?fields=email&access_token=THE_TOKEN`

I don't get the email. How facebook did such a simple thing so complicated?

Why?

**Comment by tolbon** [comment]
Why is so complicated ... cause spammer ?

**Comment by ndvbd** [comment]
Yes, of course its not me, it's for a user that is using an application, and its access token is valid and has the "email" permission.
It's an API called from a server, to facebook's servers. So I want to supply a user id and an access token.

**Comment by yguedidi** [comment]
@nadavb maybe try the debug mode and log the response?
